### PR TITLE
Add new PES SKU to Healthbot

### PR DESCRIPTION
**Pull request by arieschwartzman**
### Changelog
Add a changelog entry for this PR by answering the following questions:
  1. What's the purpose of the update?
      - [ ] new service onboarding
      - [x] new API version
      - [ ] update existing version for new feature
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify
  2. When are you targeting to deploy the new service/feature to public regions? Please provide the date or, if the date is not yet available, the month. - End of May/Start oh June
  3. When do you expect to publish the swagger? Please provide date or, the the date is not yet available, the month. End of May
  4. By default, Azure SDKs of all languages (.NET/Python/Java/JavaScript for both management-plane SDK and data-plane SDK, Go for management-plane SDK only ) MUST be refreshed with/after swagger of new version is published. If you prefer NOT to refresh any specific SDK language upon swagger updates in the current PR, please leave details with justification here.

### Contribution checklist (MS Employees Only):
- [ ] I commit to follow the [Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy) of "no breaking changes"
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist

> **Applicability**: :warning:
>
> If your changes encompass only the following scenarios, you should SKIP this section, as these scenarios do not require ARM review.
> - Change to data plane APIs
> - Adding new properties
> - All removals

Otherwise your PR may be subject to ARM review requirements. Complete the following:
- [x] Check this box if any of the following apply to the PR so that the label "ARMReview" and "WaitForARMFeedback" will be added by bot to kick off ARM API Review.  Missing to check this box in the following scenario may result in delays to the ARM manifest review and deployment.
  - Adding a new service
  - Adding new API(s)
  - Adding a new API version
    -[x] To review changes efficiently, ensure you copy the existing version into the new directory structure for first commit and then push new changes, including version updates, in separate commits. You can use OpenAPIHub to initialize the PR for adding a new version. For more details refer to the [wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version). Note that this doesn't apply if you are trying to merge a PR that was previously in the private repository.

- [x] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [x] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them.

### Breaking Change Review Checklist
If you have any breaking changes as defined in the [Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy/), request approval from the Breaking Change Review Board.

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Additional details on the process and office hours are on the [Breaking Change Wiki](https://eng.ms/docs/cloud-ai-platform/azure-core/azure-core-pm-and-design/trusted-platform-pm-karimb/service-lifecycle-and-actions-team/service-lifecycle-actions-team/apex/media/overview_breakingchanges).

NOTE: To update API(s) in public preview for over 1 year (refer to [Retirement of Previews](http://aka.ms/aprwiki))

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by arieschwartzman** [comment]
@Wzb123456789 Hi. There are some Checks that failed. Is this blocking my PR?

**Comment by Wzb123456789** [comment]
@guy-microsoft Please check this PR so that I can check bugs with the SDK owner.

**Comment by guy-microsoft** [comment]
@Wzb123456789 This looks good to me. Please proceed.

**Comment by arieschwartzman** [comment]
Hi @Wzb123456789 can you proceed to approve?

**Review by Wzb123456789 — APPROVED** [review]

File: specification/healthbot/resource-manager/Microsoft.HealthBot/stable/2023-05-01/healthbot.json [hunk 1]
```diff
@@ -0,0 +1,950 @@
...
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
```
Comment (qiaozha) [inline specification/healthbot/resource-manager/Microsoft.HealthBot/stable/2023-05-01/healthbot.json:185]: Is this operation really become a long running operation now ? I don't see any indicators in the example response which tells customers where to get the polling url .

File: specification/healthbot/resource-manager/Microsoft.HealthBot/stable/2023-05-01/examples/ResourceUpdatePatch.json [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+          "createdBy": "jack@outlook.com",
+          "createdByType": "User",
+          "createdAt": "2020-05-05T17:18:19.1234567Z",
+          "lastModifiedBy": "ryan@outlook.com",
+          "lastModifiedByType": "User",
+          "lastModifiedAt": "2020-05-06T17:18:19.1234567Z"
+        }
+      }
+    },
+    "201": {
```
Comment (qiaozha) [inline specification/healthbot/resource-manager/Microsoft.HealthBot/stable/2023-05-01/examples/ResourceUpdatePatch.json:52]: I mean this example response.

**Comment by arieschwartzman** [comment]
@raych1. Should I be able to merge the PR? It seems that I don't have authorization to merge.

**Comment by arieschwartzman** [comment]
@raych1. Should I able to merge the PR? It seems that I don't have authorization to merge.

File: specification/healthbot/resource-manager/Microsoft.HealthBot/stable/2023-05-01/healthbot.json (hunk 1 above)
Comment (guy-microsoft) [reply specification/healthbot/resource-manager/Microsoft.HealthBot/stable/2023-05-01/healthbot.json:185]: @qiaozha Right, it is. We need to add something like:
```
      "headers": {
        "Azure-AsyncOperation": "https://management.azure.com/subscriptions/12345678-1234-1234-1234-123456789098/providers/Microsoft.Healthbot/locations/eastus/operationResults/742bef43-608b-4b58-ba96-6f7a32ec5f0f?api-version=2022-08-08"
    
```  }
To the 201 response, right? (we use RPaaS, and based on the doc LRO uses Azure-AsyncOperation header).

**Comment by arieschwartzman** [comment]
@Wzb123456789  Can you help me with merging?

**Comment by raych1** [comment]
> @raych1. Should I be able to merge the PR? It seems that I don't have authorization to merge.

@arieschwartzman , can you resolve the comment from @qiaozha ?

**Comment by arieschwartzman** [comment]
@raych1. Comments resolved

**Pull request merged**